src/helpers/color.js

```diff
@@ -70,3 +68,5 @@ export const red = {
   rgb: { r: 255, g: 0, b: 0, a: 1 },
   hsv: { h: 0, s: 1, v: 1, a: 1 },
 }
+
+export default exports
```

Comment:
I was just browsing the latest release notes and inspecting the changed code and got curious about this as I couldn't find any spec about `export default exports`. It seems that this is some kind of Babel magic or a coincidence that Babel-generated code just exports a common-js implicit `module.exports` variable as the default export.

Please see the example and the generated code on the Babel REPL:
https://babeljs.io/repl#?browsers=&build=&builtIns=false&spec=false&loose=false&code_lz=KYDwDg9gTgLgBAYwgOwM7xHAvHArAbgChDRJZEV04BPbOANiMIHpm4BVZJAW2-GXgwAFgEtUcAG4BDADYiAJhXnA4MCHFTAVwlQHVoMxUhkypYTahZskaeKWgxxOAEQRuAc2fF75ZQDMpAFcZO3AHVCA&debug=false&forceAllTransforms=false&shippedProposals=false&circleciRepo=&evaluate=false&fileSize=false&timeTravel=false&sourceType=module&lineWrap=true&presets=es2015%2Creact%2Cstage-2&prettier=false&targets=&version=7.7.7&externalPlugins=

As the `exports` is not a reserved keyword, another export can be named `exports` to make this scheme to fail horribly exporting that binding, not "all the named exports" as intended by the "accidental syntax".

 Please see the generated code: https://babeljs.io/repl#?browsers=&build=&builtIns=false&spec=false&loose=false&code_lz=KYDwDg9gTgLgBAYwgOwM7xHAvHArAbgChDRJZEV04BPbOANiJPGniTXlNdToCIIAtgHNexLuQAmwAGYBDAK4AbTi1iogA&debug=false&forceAllTransforms=false&shippedProposals=false&circleciRepo=&evaluate=false&fileSize=false&timeTravel=false&sourceType=module&lineWrap=true&presets=es2015%2Creact%2Cstage-2&prettier=false&targets=&version=7.7.7&externalPlugins=

@casesandberg - I think an explicit default export of all the named exports is way better as it does not rely on Babel quirks and it is a valid piece of ES code. Are PRs welcome?

Reply:
Good catch! We are working on the next major release right now which is fully ES6, so it will not have these problems going forward. I appreciate your thoroughness, however!